contracts/extensions/CoinMachine.sol

```diff
@@ -77,10 +82,13 @@ contract CoinMachine is ColonyExtension {
     require(address(colony) == address(0x0), "extension-already-installed");
 
     colony = IColony(_colony);
+    token = colony.getToken();
```

Comment:
I added this because otherwise `uninstall` fails prior to calling `initialise`. It's not really a convention or a spec or anything, but in my mind one should be able to uninstall an extension without needing to call any functions outside of the `ColonyExtension` interface. So let's call it an informal spec. What do you think?

Reply:
I suppose I could add a check in `uninstall` which looks to see if the token is set, which is a bit clunkier but works fine.

Reply:
I see. Yeah, I prefer the added check.